src/encoded/schemas/file.json

```diff
@@ -207,6 +210,10 @@
                 "title": "Notes to tsv file",
                 "type": "string"
             }
+        },
+        "version": {
+            "type": "string",
+            "permission": "restricted_fields"
```

Comment:
Intended UX for this? May merit some further discussion on how we want to manage "versions" of files

Reply:
@willronchetti This is intended for the COLO829 Challenge data/Truth set, where presumably there might be multiple challenges and we want to keep track of which one the files came from. Open to suggestions on other ways to handle this though!